introduction-to-datasets.md

```diff
@@ -0,0 +1,103 @@
+
+## Schemas
+
+A schema defines the field names and datatypes for a dataset. Kite relies on an Apache Avro schema definition for each dataset. For example, this is the schema definition for a table using four columns from the `movies` dataset.[<sup>1</sup>](#notes)
+
+```json
+{
+  "type":"record",
+  "name":"Movie",
+  "namespace":"org.kitesdk.examples.data",
+  "fields":[
+    {"name":"id","type":"int"},
+    {"name":"title","type":"string"},
+    {"name":"releaseDate","type":"string"},
+    {"name":"imdbUrl","type":"string"}
+  ]
+}
+```
+
+The goal is to get the schema into `.avsc` format. The following links provide examples for inferring schemas from data files or Java classes.
+
+| Java API                                                      | Command Line Interface |
+| --------                                                      | ---------------------- |
+| [Inferring a schema from a Java Class][api-schema-from-class] | [Inferring a schema from a Java class][cli-schema-from-class] |
+| [Using the schema of an Avro data file][api-schema-from-data] | [Inferring a schema from a CSV file][cli-schema-from-csv] |
+
+[api-schema-from-class]: {{site.baseurl}}/Inferring-a-Schema-from-a-Java-Class.html
+[api-schema-from-data]: {{site.baseurl}}/Inferring-a-Schema-from-an-Avro-Data-File.html
+[cli-schema-from-class]: {{site.baseurl}}/cli-reference.html#obj-schema
+[cli-schema-from-csv]: {{site.baseurl}}/cli-reference.html#csv-schema
+
+## Configuration Options
+
+While you need only a URI and schema to create a dataset, you can enhance the performance of your dataset with additional configuration.
+
+### Partitions
+
+Partitions define logical categories for data storage. For example, you might most often retrieve your data using time-based queries. You can define a partitioning strategy by year, month, day, and hour. When you look for data from January 8, 2015 between 8:00 and 9:00 a.m., your search engine only has to look in the data partition `/2015/1/8/8`. By using partitions that correspond to your most common queries, your data searches run more quickly.
+
+You define your partition strategy in [JSON format][json-format] and apply it when you create your dataset. See [Partitioned Datasets][partition-strategy].
+
+[json-format]: {{site.baseurl}}/Partition-Strategy-Format.html
+[partition-strategy]: {{site.baseurl}}/Partitioned-Datasets.html
+
+
+### Column Mapping
+
+Column mapping allows you to configure how your records should be stored in HBase for maximum performance and efficiency. You define column mapping in JSON format in a data-centric way. Kite stores and retrieves the data correctly. See [Column Mapping][column-mapping].
+
+[column-mapping]: {{site.baseurl}}/Column-Mapping.html
+
+### Properties
+
+You can add custom settings to dataset properties at creation time using the `--set` option. For example, you might create the dataset _users_ using a cache size of 20 (rather than the default cache size of 10):
```

Comment:
Do you mean that we should link to a page like this one that talks about CDH tuning in general?

http://www.cloudera.com/content/cloudera/en/documentation/core/latest/topics/cdh_admin_performance.html

I found this in the POM

  <properties>
    <!-- dependency version properties -->
    <kite.version>0.16.0</kite.version>
    <kite.avro.version>1.7.5-cdh5.1.0</kite.avro.version>
    <kite.crunch.version>0.10.0-cdh5.1.0</kite.crunch.version>
    <kite.findbugs.version>2.0.1</kite.findbugs.version>
    <kite.hive.version>0.12.0-cdh5.1.0</kite.hive.version>
    <kite.oozie.version>4.0.0-cdh5.1.0</kite.oozie.version>
    <kite.spark.version>1.0.0-cdh5.1.0</kite.spark.version>

    <!-- other properties to help apps -->
    <hadoop.guava.version>11.0.2</hadoop.guava.version>
    <hadoop.log4j.version>1.2.16</hadoop.log4j.version>
    <hadoop.slf4j.version>1.6.1</hadoop.slf4j.version>
  </properties>

Is there an exhaustive list of the possible settings someplace? It would probably be a good idea to have a reference page that lists at least the most popular properties and suggestions for why a user might want to change them.